spec/backup_spec.rb

```diff
@@ -27,14 +24,18 @@
     Parity::Backup.new(
       from: "production",
       to: "staging",
-      additional_args: "--confirm thisismyapp-staging"
```

Comment:
I don't think we want to remove this line. We still want to be able to send additional arguments to  commands even if we remove configuration.